src/windows/into_file.rs

```diff
@@ -0,0 +1,16 @@
+#![cfg(feature = "std")]
+
+use std::{ffi::c_void, fs::File, os::windows::io::FromRawHandle as _};
+
+use crate::LockFile;
+
+/// Turn the [`LockFile`] into a [`std::fs::File`]; you should probably also
+/// call [`crate::lockfile_truncate`].
+/// ```
+#[doc = include_str!("../../examples/lock_preserved.rs")]
+/// ```
+impl Into<File> for &mut LockFile {
+    fn into(self) -> File {
+        unsafe { File::from_raw_handle(self.raw() as *mut c_void) }
+    }
+}
```

Comment:
[Reprise] As I said in other places: is this really required? Because this would also fall out of the scope of this feature.